LuaRules/Gadgets/unit_bomber_command.lua

```diff
@@ -511,26 +512,25 @@ function gadget:GetInfo()
 	  end
   end
   
-  local function addOrRemoveExclusion(padID)
-	  if padID == nil or not Spring.ValidUnitID(padID) then --Also check if it's a valid unit
+  local function addOrRemoveExclusion(padID, teamID)
+	  if padID == nil or not Spring.ValidUnitID(padID)then --Also check if it's a valid unit
 		return
 	  end
 
-	  local pInfo = Spring.GetPlayerInfo()
-	  local teamID = pInfo[3]
-
-	  Spring.Echo(teamID)
+	  if not Spring.GetUnitTeam(padID) == teamID then
```

Comment:
You're looking for "allyteam" here. A "team" usually corresponds to 1 player unless they do a squad merge - think of it as the same colour; a merged squad has all units (say) green so they're the same teamID, your allies have blue units so they are a different teamID, but still the same allyTeam

Reply:
Also `if not x == y` → `if x ~= y`

Reply:
I wouldn't want to exclude it for the whole ally team though right? The whole point of this is so that you can prevent your planes from landing on your allies airpads, but not prevent you allies planes from landing on said airpads?

Reply:
Yes, that is the point, but this function is about adding airpads to the exclusion (so should work on the whole "ally team") and not about deciding whose planes actually obey that exclusion (which should be just the "team" that sent it).

```suggestion
	  local allyTeamID = select(6, Spring.GetTeamInfo(teamID, false))
	  if Spring.GetUnitAllyTeam(padID) ~= allyTeamID then
```

Reply:
Ah ok, that makes sense now!